src/pages/Layout.jsx

```diff
@@ -1,11 +1,11 @@
 import React from 'react'
 import { Outlet } from 'react-router-dom'
-import Navbar from './Navbar'
+// import Navbar from './Navbar'
```

Comment:
issue: This is going to create more tech debt - you can safely delete this and we'll import it again if we need to.